glib/src/array.rs

```diff
@@ -0,0 +1,12 @@
+// Take a look at the license at the top of the repository in the LICENSE file.
+
+wrapper! {
```

Comment:
That's surprisingly short. Well, the shorter the better I guess.

Reply:
The rest I do manually for now, by casting the array.data and pushing to a vec<>. We may eventually want FromGlibContainer traits implemented? Left for another day :)

Reply:
You might want to add at least some API though. Like at least for getting the length and for getting a raw pointer to the underlying data. That should simplify your manual bindings a bit too